### PR TITLE
[12.0][MIG] web_widget_x2many_2d_matrix

### PR DESCRIPTION
**Pull request by alexey-pelykh**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
You should take into account opened PR for v11 #1014 + a current issue, which is #1084

**Comment by alexey-pelykh** [comment]
Thanks for mentioning, I've subscribed to those to keep track of them. Any other special actions to take?

**Comment by pedrobaeza** [comment]
Well, I have always missed an example embedded in this module as demo data, but it seems there's no good model for applying the 2d matrix without extra code.

**Comment by alexey-pelykh** [comment]
@pedrobaeza #1014 taken into account

**Comment by pedrobaeza** [comment]
There were a syntax error fixed here: https://github.com/OCA/web/commit/899f01fd9d9af261b2f47eb1637fe7ee3d07867f

**Comment by alexey-pelykh** [comment]
Cherry-picked https://github.com/OCA/web/commit/899f01fd9d9af261b2f47eb1637fe7ee3d07867f

**Comment by alexey-pelykh** [comment]
@tarteo @Yajo @jbeficent would like to ask for your review, if you don't mind

File: web_widget_x2many_2d_matrix/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    Show this field as value
+show_row_totals
+    If field_value is a numeric field, it indicates if you want to calculate
+    row totals. True by default
+show_column_totals
+    If field_value is a numeric field, it indicates if you want to calculate
+    column totals. True by default
+
+Example
+=======
```
Comment (yajo) [inline web_widget_x2many_2d_matrix/readme/USAGE.rst:40]: Use `~~~~~~~`

**Review by yajo — APPROVED** [review]

**Review by simahawk — APPROVED** [review]
LG

**Comment by pedrobaeza** [comment]
@Yajo the README will give error due to bad underlining, so you shouldn't merge without fixing that.

**Comment by alexey-pelykh** [comment]
Since it was merged, what should I do?

**Comment by alexey-pelykh** [comment]
(readme generator didn't error on my side for some reason)

**Comment by yajo** [comment]
The syntax it produced is not exactly wrong syntax, so it shouldn't error.

@alexey-pelykh  You can PR again to fix it.

File: web_widget_x2many_2d_matrix/readme/USAGE.rst (hunk 1 above)
Comment (alexey-pelykh) [reply web_widget_x2many_2d_matrix/readme/USAGE.rst:40]: #1106

**Comment by alexey-pelykh** [comment]
@Yajo @pedrobaeza #1106

**Pull request merged**